lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2899,16 +2899,22 @@ ParseResult FIRStmtParser::parseRefForce() {
                startTok.getLoc(),
                "expected rwprobe-type expression for force destination, got ")
            << dest.getType();
-  if (isa<RefType>(src.getType()))
+  auto srcBaseType = dyn_cast<FIRRTLBaseType>(src.getType());
+  if (!srcBaseType)
     return emitError(startTok.getLoc(),
                      "expected non-reference-type for force source, got ")
```

Comment:
maybe update the error message - it is still assuming that the type would be a reference type

Reply:
Good call, will fix thanks!

Reply:
Done, tweaked similar checks for probe/rwprobe (to be more property-friendly, mostly) and updated the messages.  Thanks!